go/cmd/vtctldclient/command/vreplication/workflow/workflow.go

```diff
@@ -60,6 +61,7 @@ func registerCommands(root *cobra.Command) {
 	delete.MarkFlagRequired("workflow")
 	delete.Flags().BoolVar(&deleteOptions.KeepData, "keep-data", false, "Keep the partially copied table data from the workflow in the target keyspace.")
 	delete.Flags().BoolVar(&deleteOptions.KeepRoutingRules, "keep-routing-rules", false, "Keep the routing rules created for the workflow.")
+	delete.Flags().Int64Var(&deleteOptions.DeleteBatchSize, "delete-batch-size", movetables.DefaultDeleteBatchSize, "The batch size to use when deleting a subset of data from the migrated tables. This is only used with multi-tenant MoveTables workflows.")
```

Comment:
Should this description not be consistent with that given for the same flag in moveables.go for the Cancel sub-command?

Reply:
Nice catch! However, this is a bit of an exception in that the context is different for `MoveTables cancel` and `Workflow delete`. It was intentional:
```
go/cmd/vtctldclient/command/vreplication/movetables/movetables.go:      cancel.Flags().Int64Var(&common.CancelOptions.DeleteBatchSize, "delete-batch-size", DefaultDeleteBatchSize, "When cleaning up the migrated data in tables moved as part of a mult-tenant workflow, delete the records in batches of this size.")
go/cmd/vtctldclient/command/vreplication/workflow/workflow.go:  delete.Flags().Int64Var(&deleteOptions.DeleteBatchSize, "delete-batch-size", movetables.DefaultDeleteBatchSize, "The batch size to use when deleting a subset of data from the migrated tables. This is only used with multi-tenant MoveTables workflows.")
```
But I'm good tweaking them if you have any suggestions or preference?

Reply:
I don't understand how they are different. As long as they are both consistent, either way of wording the flag help is fine.

Reply:
On second thought, this is somewhat misleading
```
The batch size to use when deleting a subset of data from the migrated tables
```
It makes it sound as if there's some general capability to delete a subset of data from migrated tables. So the other version is preferable.

Reply:
You need them to be exactly the same? That would be a little odd as `MoveTables cancel` is obviously for `MoveTables`. What about:
```
go/cmd/vtctldclient/command/vreplication/movetables/movetables.go:      cancel.Flags().Int64Var(&common.CancelOptions.DeleteBatchSize, "delete-batch-size", DefaultDeleteBatchSize, "When cleaning up the migrated data in tables moved as part of a multi-tenant workflow, delete the records in batches of this size.")
go/cmd/vtctldclient/command/vreplication/workflow/workflow.go:  delete.Flags().Int64Var(&deleteOptions.DeleteBatchSize, "delete-batch-size", movetables.DefaultDeleteBatchSize, "When cleaning up the migrated data in tables moved as part of a multi-tenant MoveTables workflow, delete the records in batches of this size.")
```

The only difference being we note that it's for `a multi-tenant MoveTables workflow` in the `Workflow delete` help output.

And I just realized a misspelling in there `mult-tenant` that I corrected above. :)